rj_gameplay/rj_gameplay/tactic/nmark_tactic.py

```diff
@@ -0,0 +1,115 @@
+from dataclasses import dataclass
+from typing import List, Optional
+
+import stp.action as action
```

Comment:
I believe action is imported but not used

Reply:
It is not, nice catch